code/datums/components/infective.dm

```diff
@@ -0,0 +1,13 @@
+/datum/component/infective_floor
```

Comment:
floor is kinda bad name for generic on crossed component, just infective would be enough.

Reply:
this is meant to be for puddles only, if i ever make a component for infective items it will be separate

Reply:
There's no reason to make a separate component for that, it's only a slight Initialization cost to register signals for many different objects.